code/game/objects/items/handcuffs.dm

```diff
@@ -74,12 +74,20 @@
 			else
 				to_chat(user, "<span class='warning'>You fail to handcuff [C]!</span>")
 		else
-			to_chat(user, "<span class='warning'>[C] doesn't have two hands...</span>")
+			to_chat(user, "<span class='warning'>[C] doesn't have two hands...</span>") //if you're trying to cuff yourself while one-handed, this will refer to you in the third person, which is a bit weird, but not a big enough issue to write some shitcode to keep that from happening
 
 /obj/item/restraints/handcuffs/proc/apply_cuffs(mob/living/carbon/target, mob/user, dispense = 0)
+
+	if(!istype(target)) //some sanity checks, in case apply_cuffs() gets called without attack() being called/fully run through (looking at YOU, clumsy people)
+		return

```

Comment:
This shouldn't happen. Clumsy people are `carbon` as per the proc.
Avoid sanity checks that silently return. If something like this fails we want it to scream so we know where to fix.

Reply:
A dextrous stand or a gorilla could click on themselves with fake handcuffs, no?

Reply:
It should produce no result:
```
/obj/item/restraints/handcuffs/attack(mob/living/carbon/C, mob/living/user)
	if(!istype(C))
		return
```